proto/kvrpcpb.proto

```diff
@@ -1084,6 +1090,10 @@ message PrimaryMismatch {
     LockInfo lock_info = 1;
 }
 
+message TxnLockNotFound {
+    bytes key = 1;
+}
+
```

Comment:
Do we need to add an extra message `DebugInfo` which contains some debugging info such as mvcc info here?

Reply:
Perhaps it is ok to return a single key mvcc info in the response, @MyonKeminta @ekexium @zyguan What do you think?

Reply:
In this, I does not include mvcc info in TxnLockNotFound because it seems mvcc info is not necessary for all scenes. The client should not have an assumption that it is present all the time.